packages/terra-time-input/CHANGELOG.md

```diff
@@ -13,7 +15,6 @@ Unreleased
 ------------------
 ### Added
 * Added new WDIO screenshots
-* Fixed background color issue of time components in incomplete `time-input` in Fusion theme.
```

Comment:
Whoops. Think this change wasn't intended

Reply:
looks like came in while updating the branch with latest changes.

Reply:
Actually this was an intended change.
As seen [here](https://github.com/cerner/terra-framework/pull/983/files#diff-84e7757ec3b55699f14959cc988cc871R4-R16) the change log entry was incorrectly shown as released when PR #983 was merged.
And since, this change was released on February 4, i have now corrected it here : https://github.com/cerner/terra-framework/pull/1031/commits/0c05483821789068554300b3945aaf7198b959be

Reply:
Ah nice catch! 👍